handlers/api-handler.ts

```diff
@@ -38,14 +38,23 @@ export interface ApiHandlerDefinition<B = never, Q = never, R = never>
 	scopes?: string[];
 	/**
 	 * Optional override of default AJV instance
+	 *
+	 * @deprecated Use your own AJV instance in `validators`
 	 */
 	ajv?: Ajv;
 
-	schemas: {
+	/** @deprecated Use `validators` instead */
+	schemas?: {
 		body?: JSONSchemaType<B>;
 		query?: JSONSchemaType<Q>;
 		response?: JSONSchemaType<R>;
 	};
+
+	validators?: {
+		body?: (body: any) => B;
+		query?: (query: any) => Q;
+		response?: (response: any) => R;
```

Comment:
Should these be typed as `unknown` instead? I'm not sure we want to allow a validator to mistakenly return an `any`.

Reply:
Yes, they should!